packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -70,6 +81,58 @@ export async function handler({
     const bundle = await getBundleFromS3(fhirFileName);
     const isBriefFeatureFlagEnabled = await isAiBriefEnabled(generateAiBrief, cxId);
 
+    const dedupEnabled = await isFhirDeduplicationEnabledForCx(cxId);
+    if (dedupEnabled) {
```

Comment:
feels like this part of the code can be factored out into a reusable function - but not a blocker on this PR

Reply:
its gonna get moved around since this is really just to store the deduped MR for now
Lets address it on the next one when we actually embed this in the flow